docs/core/deploying/trimming-options.md

```diff
@@ -123,3 +123,37 @@ Symbols will normally be trimmed to match the trimmed assemblies. You can also r
     Remove symbols from the trimmed application, including embedded PDBs and separate PDB files. This applies to both the application code and any dependencies that come with symbols.
 
 The SDK also makes it possible to disable debugger support using the property `DebuggerSupport`. When debugger support is disabled, trimming will remove symbols automatically (`TrimmerRemoveSymbols` will default to true).
+
+## Trimming framework library features
+
+Several feature areas of the framework libraries come with linker directives that make it possible to remove the code for disabled features.
+
+- `<DebuggerSupport>false</DebuggerSupport>`
+
+    Remove dependencies that enable better debugging experiences. This will also [remove symbols](#Removing-symbols).
+
+- `<EnableUnsafeBinaryFormatterSerialization>false</EnableUnsafeBinaryFormatterSerialization>`
+
+    Remove BinaryFormatter serialization support. See [BinaryFormatter serialization methods are obsolete](https://docs.microsoft.com/en-us/dotnet/core/compatibility/corefx#binaryformatter-serialization-methods-are-obsolete-and-prohibited-in-aspnet-apps) for more information.
+
+- `<EnableUnsafeUTF7Encoding>false</EnableUnsafeUTF7Encoding>`
+
+    Remove insecure UTF-7 encoding code. See [UTF-7 code paths are obsolete](https://docs.microsoft.com/en-us/dotnet/core/compatibility/corefx#utf-7-code-paths-are-obsolete) for more information.
+
+- `<EventSourceSupport>false</EventSourceSupport>`
+
+    Remove EventSource related code or logic.
+
+- `<HttpActivityPropagationSupport>false</HttpActivityPropagationSupport>`
+
+    Remove dependencies related to diagnostics support for System.Net.Http.
+
+- `<InvariantGlobalization>true</InvariantGlobalization>`
+
+    Remove globalization specific code and data. See [Invariant mode](https://docs.microsoft.com/en-us/dotnet/core/run-time-config/globalization#invariant-mode) for more information.
+
+- `<UseSystemResourceKeys>true</UseSystemResourceKeys>`
+
+    Remove localizable resources for system assemblies.
```

Comment:
We don't localize these resources at all 😞, so maybe we should steer clear of "localizable resources". How about?

```suggestion
    Strips exception messages for System.* assemblies. When an exception is thrown from a System.* assembly, the message will be a simplified resource ID instead of the full message.
```